test/runtests.jl

```diff
@@ -57,8 +57,10 @@ function stable_sprand(r::AbstractRNG, m::Integer, n::Integer, density::Abstract
   sparse(I, J, V)
 end
 
-for test in tests
-    @testset "$test" begin
-        include(test)
+withenv("FORCE_COLOR"=>"X") do  # github.com/JuliaPlots/UnicodePlots.jl/issues/134
```

Comment:
Didn't know this is possible, thanks for the fix!

Reply:
That's a `Crayons` feature:
https://github.com/KristofferC/Crayons.jl/blob/2619561156a02fdf739c8146e09a96ceb176f43c/src/crayon.jl#L1